debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_configure:
+	./autogen.sh
```

Comment:
The annoying bit is that it sets up the empty m4 directory. When I saw that earlier I didn't see how to easily get rid of that. So you always get that silly warning. :/